### PR TITLE
Cc to mailscanner

### PR DESCRIPTION
**Pull request by denaldd**

### CODE REVIEW

File: modules/Settings/MailScanner/core/MailScannerRule.php [hunk 1]
```diff
@@ -369,21 +383,22 @@ public function update() {
...
 			$adb->pquery(
-				'UPDATE vtiger_mailscanner_rules SET scannerid=?,fromaddress=?,toaddress=?,subjectop=?,subject=?,bodyop=?,body=?,matchusing=?,assign_to=? WHERE ruleid=?',
+				'UPDATE vtiger_mailscanner_rules SET scannerid=?,fromaddress=?,toaddress=?,subjectop=?,subject=?,bodyop=?,body=?,matchusing=?,assign_to=?,cc=? WHERE ruleid=?',
 				array($this->scannerid, $this->fromaddress, $this->toaddress, $this->subjectop, $this->subject,
-				$this->bodyop, $this->body,$this->matchusing,$this->assign_to,$this->ruleid)
+				$this->bodyop, $this->body,$this->matchusing,$this->assign_to, $this->cc, $this->ruleid)
 			);
 		} else {
-			$this->sequence = $this->__nextsequence();
+			$this->sequence = $this->__nextsequence();var_dump($this->cc);
```
Comment (joebordes) [inline modules/Settings/MailScanner/core/MailScannerRule.php]: eliminate this dump

Comment (denaldd) [reply modules/Settings/MailScanner/core/MailScannerRule.php]: done

**Comment by joebordes** [comment]
Looks good to me.
Apply our [code formatting rules](http://corebos.com/documentation/doku.php?id=en:devel:developmentguidelines#code_formatting_and_structure) and for the next time, our [commit message guidelines](http://corebos.com/documentation/doku.php?id=en:devel:developmentguidelines#commit_guidelines) also

**Comment by joebordes** [comment]
ah, sorry, I didn't see the actual commit message are correct
wonderful!!  thank you!! :-D

**Comment by joebordes** [comment]
thanks!!

**Pull request closed without merging**